api/net/Areas/Services/Controllers/ReportController.cs

```diff
@@ -84,17 +84,15 @@ public IActionResult FindById(int id)
     /// <returns></returns>
     [HttpGet("{id}/content")]
     [Produces(MediaTypeNames.Application.Json)]
-    [ProducesResponseType(typeof(IEnumerable<Models.Content.ContentModel>), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(Elastic.Models.SearchResultModel<API.Areas.Services.Models.Content.ContentModel>), (int)HttpStatusCode.OK)]
```

Comment:
We now return the Elastic results instead of extracting only the content.  This will enable future enhancements in reports to more easily align to elasticsearch.